src/main/scala/lisa/automation/kernel/SimplePropositionalSolver.scala

```diff
@@ -165,4 +168,32 @@ object SimplePropositionalSolver {
     r4
   }
 
+  case object Trivial extends ProofStepWithoutBot with ProofStepWithoutBotNorPrem(-1) {
+    override val premises: Seq[Int] = Seq()
+    def asSCProof(bot: Sequent, currentProof: Library#Proof): ProofStepJudgement = {
+      ProofStepJudgement.ValidProofStep(SCSubproof(solveSequent(bot)))
+    }
+    def asSCProof(bot: Sequent, premises: Seq[Int], currentProof: Library#Proof): ProofStepJudgement = {
+
+      val sp = SCSubproof(
+        {
+          val premsFormulas = premises.map(p => (p, sequentToFormula(currentProof.getSequent(p))))
+          val initProof = premsFormulas.map(s => Rewrite(() |- s._2, s._1)).toList
+
+          println("I was called! " + initProof.length)
```

Comment:
There's a testing `println` left in the `SimplePropositionalSolver` :) Seems to be the only one I've been able to find.

Reply:
Nice catch :) Solved